.buildkite/scripts/upload_backline_to_acceptance.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/shared.sh
+
+echo "--- :drum_with_drumsticks: Uploading hab-backline to acceptance" 
+
+acceptance_channel() {
+  if is_fake_release; then
+    echo "$(get_release_channel)"
+  else
+    echo "stable"
+  fi
+}
+
+backline_ident="$(get_backline_ident x86_64-linux)"
+backline_artifact="$(get_backline_artifact x86_64-linux)"
+
+# Ensure we have the package in our local artifact cache
+hab pkg install "$backline_ident"
+
+# The above pkg install is executed in the context of the buildkite user
+# so the artifact (and public signing keys) will be cached in the users
+# home, rather than the system cache (/hab). If the below breaks with a:
+#
+# Invalid value for '<HART_FILE>...': File: 'XXXX.hart' cannot be found, 
+#
+# then it is likely that something has changed with how Hab caches files,
+# or the user this is executed as.
+hab pkg upload -u https://bldr.acceptance.habitat.sh -c "$(acceptance_channel)" \
```

Comment:
In the testing I did no additional credentials were needed, I think due to how acceptance is set up.  

Short-term I think we're ok. I would probably be good to look at getting a separate set of credentials for acceptance so we don't break things later, but that will probably involve a change to how acceptance is deployed as well.

Reply:
One thing that might be nice here is to explicitly add the `--auth` flag, and use the appropriate environment variable as the value. That way we can immediately see exactly which token is being used, which would help with debugging in the future.

It'd be nice to nail down exactly what the auth situation is now, though, just so we have as much information as possible.

(And just as an aside, I find it to greatly enhance the readability of these scripts to use the long form of the options (that is, `--url` instead of `-u`, and `--channel` instead of `-c`). The general rule of thumb I use is "short flags when I'm typing commands interactively for convenience, and long flags when I'm scripting for long-term readability and more self-documenting code".)

Reply:
I've updated this to add the long forms of the flags.  I've held off on the --auth for now to get @scotthain input, that makes me a little nervous that we might leak our token if we go that route.

Reply:
I'm fine with using `--auth`, we do a similar thing here: https://github.com/habitat-sh/habitat/blob/master/.buildkite/scripts/shared.ps1#L60 on the windows side of things.